doc/refman/RefMan-pro.tex

```diff
@@ -103,6 +103,14 @@ \section{Switching on/off the proof editing mode}
 This command is available in interactive editing proof mode to give up
 the current proof and declare the initial goal as an axiom.
 
+\variant {\tt Unset Keep Admitted Variables}
+
+By default in sections, an {\tt Admitted} lemma is generalized by all
+variables in the context (or by those in the {\tt Proof Using}
+expression if it exists) at the end of the section. Unsetting this
+option generalizes {\tt Admitted} lemmas by only those variables used
+in its type and partial proof.
```

Comment:
This seems like a corner case where async might do the wrong thing.
OTOH I think it's disabled for Admitted.

Reply:
This documentation doesn't seem to be reflecting the current behavior:
```
Unset Keep Admitted Variables.

Section Foo.

Variable x : nat.

Lemma foo : True.
Proof.
generalize x.
Show Proof. (* x is used in the partial term *)
Admitted.
End Foo.
About foo. (* no hypothesis *)
```

Please fix or clarify if it is a bug.

Reply:
I forgot what tests I did to figure out what this option does so I'm just dropping it.